ModularTegustation/tegu_items/backstreets/random_rooms/medium/north.dm

```diff
@@ -14,3 +14,9 @@
 	template_height = 6
 	stock = 3
 	room_type = "backstreets_medium_north"
+
+ /datum/map_template/random_room/backstreets/medium_north/workshop_mn
```

Comment:
Remove the space at the back here, DM is based heavily on tabs, and spaces screw it up

Reply:
found the space, removed